docs/io/visualization/interaction_radius_plot.ipynb

```diff
@@ -0,0 +1,7286 @@
+{
```

Comment:
<p>You didn't have to reuse the SDEC text verbatim 😅</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>